### PR TITLE
Update templates

### PR DESCRIPTION
**Pull request by SansSeryph**

### CODE REVIEW

File: template/provision/group_vars/all.erb [hunk 1]
```diff
@@ -5,10 +5,12 @@ use_sudo: true
...
+ruby_download_location: https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.1.tar.gz
+bundler_version: 2.1.4
 
-# passenger
-passenger_webserver: apache
-passenger_pkgs_state: present
+# Other stuff
+letsencrypt_email: 
+nodejs_version: 13.x
+ssl_enabled: true
```
Comment (Nonsense331) [inline template/provision/group_vars/all.erb:16]: Oh nice can you add `postgresql_version: 11.7` here too? right now we default to 9.4 and its pretty bad

File: template/provision/templates/application.yml.template [hunk 2]
```diff
@@ -4,24 +4,24 @@
 # These environment variables are available to all environments, and can be secret or not:
 
 # These are secret and can be changed per environment easily by using subspace vars <env> --edit
-SECRET_KEY_BASE: {{SECRET_KEY_BASE}}
-AWS_SECRET_KEY: {{AWS_SECRET_KEY}}
+# SECRET_KEY_BASE: {{SECRET_KEY_BASE}}
```
Comment (Nonsense331) [inline template/provision/templates/application.yml.template:7]: should we leave this one uncommented as all rails projects need it?

Comment (Nonsense331) [reply template/provision/templates/application.yml.template:7]: actually thats totally not true... i think i'm fine with it left out

**Comment by Nonsense331** [comment]
you should rebase instead of merging master into feature branches :D but this is great 🛳

**Comment by SansSeryph** [comment]
Oops, I didn't realize I'd done that! I'll be more mindful of that in the future :)

**Pull request merged**